image/scripts/dev/installLatestK8sCesSetup.sh

```diff
@@ -23,6 +23,12 @@ kube_ctx_name=${15}
 # set environment for helm and kubectl
 export KUBECONFIG="${HOME}/.kube/$kube_ctx_name"
 
+if command -v kubens &> /dev/null
+then
+    echo "set k8s namespace via kubens"
+    kubens "$CES_NAMESPACE"
```

Comment:
It could be more likely, that only "kubectl" is available and not "kubens".
To set the namespace via "kubectl" we could use: `kubectl config set-context --current --namespace=$CES_NAMESPACE`

Reply:
Great addition to the ecosystem here! :+1: 

My only $.02 to @kahoona77's comment ist that it is best practice to use curly braces for shell variables: 
`"${CES_NAMESPACE}"`